app/views/lti_deployment/choose_course.html.erb

```diff
@@ -1,13 +1,29 @@
+<%= javascript_tag nonce: true do %>
+  document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Please fix the indentation for this code (clearly the copy-and-paste dropped the formatting)